pkgs/tools/system/s-tui/default.nix

```diff
@@ -1,19 +1,23 @@
-{ stdenv, pythonPackages }:
+{ stdenv, python3Packages, glibcLocales }:
 
-pythonPackages.buildPythonPackage rec {
+python3Packages.buildPythonPackage rec {
   pname = "s-tui";
   version = "1.0.0";
 
-  src = pythonPackages.fetchPypi {
+  src = python3Packages.fetchPypi {
     inherit pname version;
     sha256 = "0r5yhlsi5xiy7ii1w4kqkaxz9069v5bbfwi3x3xnxhk51yjfgr8n";
   };
 
-  propagatedBuildInputs = with pythonPackages; [
+  propagatedBuildInputs = with python3Packages; [
     urwid
     psutil
   ];
 
+  checkInputs = [
+    glibcLocales
+  ];
```

Comment:
A better way to fix the error that doesn't need `glibcLocales` is
```
  LC_ALL = "en_US.UTF-8";
```

Reply:
Updated.